README.md

```diff
@@ -34,6 +34,11 @@ Create a progress bar.
 - __reverse:__ (default: `'false'`) reverses the progress bar direction
 - __class:__ (default: `''`) set additional classes on the element
 
+## toolbar(opts?)
+Create a toolbar.
+- __innerContent:__ (default: `''`) set the innerContent of the toolbar
```

Comment:
As with the other APIs, we tend to pass an optional final argument for any child nodes; makes for a more natural API imo